Content.Shared/TapeRecorder/Components/TapeCassetteComponent.cs

```diff
@@ -0,0 +1,43 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.TapeRecorder.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(true)]
+[Access(typeof(SharedTapeRecorderSystem))]
+public sealed partial class TapeCassetteComponent : Component
+{
+    /// <summary>
+    /// A list of all recorded voice, containing timestamp, name and spoken words
+    /// </summary>
+    [DataField]
+    public List<TapeCassetteRecordedMessage> RecordedData = new();
+
+    /// <summary>
+    /// The current position within the tape we are at, in seconds
+    /// Networked for client side prediction
+    /// </summary>
+    //Annoyingly im seeing a 8 - 10% discrepency between client server frame times - as such this needs to be auto networked
+    [DataField]
```

Comment:
Server and client frame times are exactly the same.

Reply:
No, both server and client increment CurrentPosition by frameTime in the Update method.
After ~20 seconds i noticed that the client CurrentPosition is around 0.5 seconds slower than the server.
![image](https://github.com/space-wizards/space-station-14/assets/11758391/400d07c3-2bad-42ac-9538-503f0d485aef)

Reply:
If it's not being predicted properly that will happen.